pkg/hooks/structs/structs.go

```diff
@@ -1,28 +1,13 @@
 package structs
 
 // ConnID is a conversion of the following C-Struct into GO.
-//
```

Comment:
Along with the above comment please also add the the following commented code which depicts the c code.

Reply:
Ok then I will restore enums and structs.go files to their original state.